jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/controller/UserController.java

```diff
@@ -533,6 +529,48 @@ public JsonResponse sendEmailConfirmation(@RequestParam("id")long id){
         return new JsonResponse(JsonResponseStatus.SUCCESS);
     }
 
+    @ResponseBody
+    @RequestMapping(value = "/user/{userID}/groups", method=RequestMethod.GET)
+    public JsonResponse userGroups(@PathVariable("userID") long userID){
+        try {
+            JCUser jcUser = userService.get(userID);
+            List<Long> groupsIDs = jcUser.getGroupsIDs();
+            return new JsonResponse(JsonResponseStatus.SUCCESS, groupsIDs);
+        } catch (Exception e) {
+            return new JsonResponse(JsonResponseStatus.FAIL);
+        }
+    }
+
+    @ResponseBody
+    @RequestMapping(value = "/user/{userID}/group/{groupID}", method=RequestMethod.POST)
+    public JsonResponse addUserToGroup(@PathVariable long userID, @PathVariable long groupID){
```

Comment:
What about authorization? It looks like any registered user can invoke these methods.